pkgs/tools/package-management/nix-template/default.nix

```diff
@@ -8,25 +9,35 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nix-template";
-  version = "0.1.1";
+  version = "0.1.4";
 
   src = fetchFromGitHub {
+    name = "${pname}-${version}-src";
     owner = "jonringer";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-A1b/fgSr27sfMDnTi4R3PUZfhAdLA5wUOd4yh9/4Bnk=";
+    sha256 = "sha256-kNFhSfHUYBUOCXoD6m7thMho4tOIpRHfHGcsW8FTgkc=";
   };
 
-  cargoSha256 = "sha256-resyY/moqLo4KWOKUvFJiOWealCmcEsLFgkN12slKN0=";
+  cargoSha256 = "sha256-7PthFLCEt+E/Gx5//aulHYYBKZqapNEWKtKfRlDr3Pw=";
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+    pkg-config
+  ];
 
-  nativeBuildInputs = [ makeWrapper pkg-config ];
   buildInputs = [ openssl ]
     ++ lib.optional stdenv.isDarwin Security;
 
   # needed for nix-prefetch-url
   postInstall = ''
     wrapProgram $out/bin/nix-template \
       --prefix PATH : ${lib.makeBinPath [ nix ]}
+
+    installShellCompletion --bash --cmd nix-template <($out/bin/nix-template completion bash)
+    installShellCompletion --zsh --cmd nix-template <($out/bin/nix-template completion zsh)
+    installShellCompletion --fish --cmd nix-template <($out/bin/nix-template completion fish)
```

Comment:
```suggestion
    installShellCompletion --cmd nix-template \
      --bash <($out/bin/nix-template completions bash) \
      --fish <($out/bin/nix-template completions fish) \
      --zsh <($out/bin/nix-template completions zsh)
```

Reply:
I don't know if that's strictly better.

Had issues trying to use installShellCompletion in the past, and would rather not complicate the cli usage. Then again, I used to not be able to do a redirect, so maybe now it's better

Reply:
i tried it and it works 
if you decide to keep the current on the you have to change `..template completion` to `..template completions`

Reply:
oh, oops! Darn s :)

Reply:
I'll keep it for now, but thanks for the correction :)